mobify/progressive-web-scaffold
# Update sdk to 0.12.0

## Changes
- Address breaking changes introduced in the SDK 0.12.0
- Remove all reference to `IconSprite`
- Fix all `import {Icon}` to read `import Icon`
- Note that we should replace importing the SVG sprite with fetching it as an external asset via Ajax instead

## How to test-drive this PR
- Run `npm update progressive-web-sdk`
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Verify that all icons work, and the site works as intended
